core/_variables.scss

```diff
@@ -14,9 +14,13 @@ $base-line-height: 1.5;
 $heading-line-height: 1.2;
 
 // Other Sizes
+$space-xs: 0.5em;
```

Comment:
I agree that the spacing system breaks down. I typically end up writing these variables anyhow, so I would be okay with this change. 

Thoughts on using BEM like syntax for the variables names and being more verbose? Something like:

```scss
$spacing--extra-small: 0.5em;
$spacing--small: 0.75em;
$spacing--medium: 1em;
```